modules/ROOT/pages/asm-secret-group-creation-task.adoc

```diff
@@ -166,14 +166,12 @@ Add a list of certificates that were revoked by certificate authorities before t
 
 == Edit a Secret Group
 
-When you edit a secret group, other users are locked out while you edit the necessary security objects. If a user doesn't end an editing session by either finishing or canceling it, the *Edit* button is disabled for other users. If you click *Cancel Edit*, the open editing session is closed, and the *Edit* button becomes available, even if the session was started by a different user. 
+Edit a secret group to add secret types such as Keystore, Truststore, and Certificates.
 
+. In the *Secret Groups* list view, select the secret group you want to edit, and click *Edit*. 
+. Make your changes as required, then click *Finish.*
```

Comment:
Can we remove this line? 'Make your changes as required, then click *Finish.*'. The finish button will not exist after the release. Intead, can we add this line - 'Make your changes as required, then click *Save*, to save each secret type.'

Reply:
Revised as:

. Make changes to the secrets as required.

As you create or save changes to a secret, the changes are applied immediately. The updated secret group is immediately available the next time the secret group is accessed or when deploying an application.